### PR TITLE
Update some python ports

### PR DESCRIPTION
**Pull request by catap**
###### Type(s)

- [x] bugfix
- [x] enhancement

###### Tested on

macOS 12.6.8 21G725 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by catap** [comment]
@reneeotten I need your review and help with attempt to enable tests on some ports where I've disable it.

I simple can't figure out how to bypass all required options and keep things works.

If you have time and will do it, feel free to push to the branch any fixes.

**Comment by reneeotten** [comment]
I'll take a look tonight

**Comment by reneeotten** [comment]
@catap the only one I could fix was `py-pytest-trio`, the other ones are not easily fixed I am afraid... I corrected a few minor details in the Portfiles here and there but overall it looks good to me.

**Comment by catap** [comment]
@reneeotten thanks. I not sure that we should keep `test.run yes` at `py-trio` if we know that its failing.

**Comment by reneeotten** [comment]
True... good point, we shouldn't. Can you make that change, I am not  in front of a Mac for a while.

**Comment by catap** [comment]
@reneeotten sure, here it is.

File: python/py-blinker/Portfile [hunk 1]
```diff
@@ -18,19 +20,37 @@ long_description \
...
+homepage            https://blinker.readthedocs.io
 
-checksums           sha256  471aee25f3992bd325afa3772f1063dbdbbca947a041b8b89466dc00d606f8b6 \
-                    rmd160  54a649ae37e54924ebfe75149c3d19b0d25aa1a4 \
-                    size    111476
+checksums           sha256  4afd3de66ef3a9f8067559fb7a1cbe555c17dcbe15971b05d1b625c3e7abe213 \
+                    rmd160  549652ed6e69930cfc52fe6525773f03cdbb52c2 \
+                    size    28699
 
 python.versions     27 35 36 37 38 39 310 311
```
[the remark this answers is not in the document]
Comment (pmetzger) [reply python/py-blinker/Portfile:29]: @catap Please do remove them unless there's good reason not to.

**Comment by catap** [comment]
@herbygillot, @reneeotten, @mascguy shall it be merged?

**Comment by catap** [comment]
BTW I do have the next PR which removes ~dozen of unrequired `py37` subports and adds a few `py311`, but it depends on this one.

**Comment by catap** [comment]
Thanks, Renee!

**Pull request merged**